client/controllers/articles.coffee

```diff
@@ -71,22 +86,44 @@ Template.articles.events
       if not closeRow
         #TODO: Display event source details.
       ###
+  "click #event-sources-table tbody tr": (event, templateInstance) ->
+    event.preventDefault()
+    templateInstance.selectedSourceId.set @_id
+    $(event.currentTarget).parent().find('tr').removeClass("details-open")
+    $(event.currentTarget).addClass("details-open")
   "click .open-source-form": (event, template) ->
     Modal.show("sourceModal", {userEventId: template.data.userEvent._id})
 
+
 Template.articleSelect2.onRendered ->
   $input = @$("select")
   options = {}
-
   if @data.multiple
     options.multiple = true
-
   $input.select2(options)
-
   if @data.selected
     $input.val(@data.selected).trigger("change")
   $input.next(".select2-container").css("width", "100%")
+  "click tr": (event, templateInstance) ->
+    event.preventDefault()
+    templateInstance.selectedSourceId.set @_id
+  "click .open-source-form": (event, template) ->
+    Modal.show("sourceModal", {userEventId: template.data.userEvent._id})
 
 Template.articleSelect2.onDestroyed ->
   templateInstance = Template.instance()
   templateInstance.$("#" + templateInstance.data.selectId).select2("destroy")
+
+Template.articleSelect2.helpers
```

Comment:
I think this was accidentally reintroduced. The onRendered function is now used to initialize the select2.